### ISSUE TITLE
BUG: GCC 10.2.0 auto-enables -fno-common which block multiple definitions

### ISSUE THREAD

**Issue by Eonfge** [ezQuake/ezquake-source#386]
Please search for existing issues and check for potential duplicates before filing yours.

**ezQuake version:**
3.2.1

**OS/device including version:**
Flatpak SDK 20.08
GCC 10.2.0
Target: x86_64-unknown-linux-gnu

**Describe the bug**
When compiling, I get the following error:
```
multiple definition of `temp_entities'; .ezquake/cmodel.o:/run/build/ezquake-source/protocol.h:468: first defined here
```
https://flathub.org/builds/#/builders/48/builds/3061/steps/6/logs/stdio

**Workaround**
```
LDFLAGS='-Wl,--allow-multiple-definition'
```

Previously, I mentioned this in #233 but that was just a red harring

**Comment by meag** [comment]
Thanks - I fixed this https://github.com/ezQuake/ezquake-source/commit/b174271b7d193bf03a7270445e710984b7325b62 but will include in 3.2.2 fixes

**Comment by meag** [comment]
Fixed in 3.2.2 (release soon)
